### ISSUE TITLE
Melhorar CONTRIBUTING.md

### ISSUE THREAD

**Issue by vitorp** [vitorp/matricula-web-api#1]
Criar documentação e padrões de contribuição básicos de open source para o projeto no arquivo CONTRIBUTING.md. Estou pensando em coisas básicas como um template simples de pull request e explicações de como desenvolver para o projeto. O pull request dessa issue ser feito para o branch "open-source".

**Comment by renanreboredo** [comment]
Nessas informações para desenvolver estaria o setup do projeto né?

**Comment by vitorp** [comment]
Sim. Já tem aquelas de docker lá, mas acho que ele ta muito complexo. Então queria simplificar ele e tambem dar uma opção pra configurar localmente normal que ai precisaria do #3
